.circleci/config.yml

```diff
@@ -362,6 +367,28 @@ workflows:
           requires:
             - prep-build-ts-migration-dashboard
 
+  locales_only:
+    when:
+      matches:
+        pattern: /^l10n_crowdin_action$/
+        value: << pipeline.git.branch >>
+    jobs:
+      - prep-deps
+      - get-changed-files-with-git-diff:
+          <<: *exclude_develop_master_rc
```

Comment:
`exclude_develop_master_rc` is an alias defined on line 56 that makes this step not run on certain branches.  But you're already only running this workflow only on branch `l10n_crowdin_action`, so this is unneccessary.